main.go

```diff
@@ -42,18 +46,24 @@ func main() {
 			log.Fatalf("expected line to have 2 elements, got %d for %v", len(line), line)
 		}
 
-
 		quiz = append(quiz, item{
 			Question: line[0],
 			Answer:   line[1],
 		})
 	}
 
-	resultsJSON, err := json.MarshalIndent(quiz, "", "    ")
+	possibleScore := len(quiz)
+	actualScore := 0
 
-	if err != nil {
-		log.Fatal(err)
+	for i, item := range quiz {
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Printf("Problem #%v: %s = ", i, item.Question)
+		text, _ := reader.ReadString('\n')
+
+		if strings.TrimSpace(text) == item.Answer {
```

Comment:
Okay to inline the trimming of the variable? Or would you rather name it?

Reply:
inline is fine.  You're not doing anything else with it.